liqd_product/assets/scss/components/nav.scss

```diff
@@ -0,0 +1,44 @@
+.nav {
```

Comment:
this classname is already in use in comments (from bootstrap). Can we use something else?